ex_28.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+#step1: create an empty list
+new_list = []
+
+#step2: create a variable equal to the number of values I need input for
+num = 5
```

Comment:
Since this value should never change during the execution of this program, it should be a constant.